### PR TITLE
Mendix Cloud only uses PostgreSQL

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

**Review by mbyregowMx — APPROVED** [review]

File: content/developerportal/deploy/mendix-cloud-deploy.md [hunk 1]
```diff
@@ -69,6 +69,12 @@ By default, apps are deployed to the Mendix Cloud **v4**. Features which are cov
 
 In some circumstances, you might find that the two or three standard environments you get with a licensed app are not sufficient. In this case you can request a Mendix Cloud v4 node with *Flexible Environments*. With Flexible Environments you can specify how many environments you want in your node and you can decide what they should be called.
 
+### 1.4 Database
+
+Apps deployed to the Mendix Cloud are configured to use a PostgrSQL database. It is not possible to configure your app to use an alternative database if it is deployed to the Mendix Cloud.
```
Comment (jeohist) [inline content/developerportal/deploy/mendix-cloud-deploy.md:74]: Typo in PostgreSQL :)

Comment (MarkvanMents) [reply content/developerportal/deploy/mendix-cloud-deploy.md:74]: Aaargh.
I think there is something wrong with my `e` key today. I've typed postgreSQL so many times and each time I missed the e. I thought I'd caught all of them - but not the most important.
Thanks for catching that!

Comment (MarkvanMents) [reply content/developerportal/deploy/mendix-cloud-deploy.md]: Updated in https://github.com/mendix/docs/pull/3703/commits/45e2540cc5ed8d8ea9148a6af227189ed76387ae

**Comment by MarkvanMents** [comment]
Approved over Slack.

**Pull request merged**